BREAKING.md

```diff
@@ -17,18 +15,23 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 - Avoid using code formatting in the title (it's fine to use in the body).
 - To explain the benefit of your change, use the [What's New](https://fluidframework.com/docs/updates/v1.0.0/) section on FluidFramework.com.
 
-# 3.0.0
+# 2.0.0-internal-1.1.0
```

Comment:
Isn't it `internal.1.1`, not `internal-1.1`?

Reply:
Also... should this section (for `next` stuff) be about `2.0.0-internal-2.0.0`?  i.e. the next major bump?

Reply:
> Also... should this section (for `next` stuff) be about `2.0.0-internal-2.0.0`? i.e. the next major bump?

Yes, everything under the `2.0.0-internal-1.0.0 Upcoming changes` should be actionable items that you can take for to prepare for breaking changes in a future major release

[More in-depth explanation](https://github.com/microsoft/FluidFramework/wiki/Communicating-breaking-changes#breaking-changes-vs-upcoming-changes)

Reply:
Ok I see now that minor bumps get Upcoming sections.  I would lean towards using the major version in the H1 heading.

Or maybe `# 2.0.0-internal.1.x` or `# 2.0.0-internal.1.0.0 through 2.0.0-internal.1.1.0` and update it each release.  I'd prefer the former (`1.x`)